src/app_service/service/wallet_account/service.nim

```diff
@@ -555,6 +553,12 @@ QtObject:
             tokens = map(tokensDetailsObj.getElems(), proc(x: JsonNode): WalletTokenDto = x.toWalletTokenDto())
             tokens.sort(priorityTokenCmp)
             data.accountsTokens[wAddress] = tokens
+
+            # set assetsLoading to false once the tokens are loaded
```

Comment:
@Khushboo-dev-cpp please make a private proc for setting assets loading to true/false and use it here and in the line 589 below.

`proc updateAssetsLoadingState(self: Service, loading: bool) =`

Reply:
thanks for pointing that out, done :)